### PR TITLE
SUBMARINE-545. Changes of top level K8s README to make it easier to read

### PR DESCRIPTION
**Pull request by wangdatan**
### What is this PR for?
* Changes of top level K8s README.

### What type of PR is it?
[Document]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-545

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes

### CODE REVIEW

**Comment by wangdatan** [comment]
@jiwq, @tangzhankun can you help to review it? Mostly it is cosmetic changes. And it changed text about how to set up local K8s cluster a bit.

File: docs/userdocs/k8s/README.md [hunk 1]
```diff
@@ -22,19 +22,40 @@ Submarine for K8s supports distributed TensorFlow and PyTorch.
 
 Submarine can run on K8s >= 1.14, supports features like GPU isolation.
 
+We have validated Submarine on the following versions:
+
+[FIXME]: is it accurate?
```
Comment (wangdatan) [inline docs/userdocs/k8s/README.md:27]: @tangzhankun , @jiwq , @pingsutw : What we should put for supported versions?

Comment (jiwq) [reply docs/userdocs/k8s/README.md:27]: LGTM, I had verified 14/15/16 version.

File: docs/userdocs/k8s/README.md [hunk 2]
```diff
@@ -22,19 +22,40 @@ Submarine for K8s supports distributed TensorFlow and PyTorch.
...
+### Model Training With Submarine Python SDK
+
+Submarine Python SDK can runs on any machine and it will talk to Submarine Server via REST API. So you can install Submarine Python SDK on your laptop, a gateway machine, your favorite IDE (like PyCharm/Jupyter, etc.).
 
 #### Prepare Python Environment to run Submarine SDK
 
-Submarine SDK assumes Python3.7+ is ready.
+First of all
+
+Submarine SDK requires Python3.7+.
```
Comment (pingsutw) [inline docs/userdocs/k8s/README.md:58]:
```suggestion
Submarine SDK requires Python3.5+.
```
We support Python 3.5, 3.6, 3.7.
Tensorflow is not compatible with python3.8, so if they use deepfm or other predefined ctr models, they will get the error.

File: docs/userdocs/k8s/README.md [hunk 3]
```diff
@@ -46,33 +67,37 @@ python3 virtualenv-16.0.0/virtualenv.py venv
...
-#### With Submarine SDK (Recommended)
+#### Install Submarine SDK
 
-- Install SDK from pypi.org
+**Install SDK from pypi.org (recommended)**
 
 Starting from 0.4.0, Submarine provides Python SDK. Please change it to a proper version needed.
 
 ```bash
 pip install submarine-sdk==0.4.0
```
Comment (lowc1012) [inline docs/userdocs/k8s/README.md:77]: pip install pysubmarine==0.4.0

Comment (tangzhankun) [reply docs/userdocs/k8s/README.md:77]: @pingsutw @wangdatan which name should we use? "pysubmarine" or "submarine-sdk"?

Comment (pingsutw) [reply docs/userdocs/k8s/README.md:77]: I'm ok with these two names. I prefer "pysubmarine".
just like arrow SDK named "pyarrow"

Comment (pingsutw) [reply docs/userdocs/k8s/README.md:77]: I've upload `apache-submarine` to PyPi
https://pypi.org/project/apache-submarine/

**Review by xunliu — COMMENTED** [review]
This PR has been waiting for too long. I will merge it first, and then create a other jira for modification if there is a problem.

**Pull request closed without merging**